packages/perseus/src/components/math-output.tsx

```diff
@@ -11,22 +10,33 @@ import TexWrangler from "../tex-wrangler";
 
 const ModifyTex = TexWrangler.modifyTex;
 
-class MathOutput extends React.Component<any, any> {
-    static propTypes = {
-        value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-        className: PropTypes.string,
-        labelText: PropTypes.string,
-        onFocus: PropTypes.func,
-        onBlur: PropTypes.func,
-    };
-
-    static defaultProps: any = {
+type Props = {
+    value: string | number;
+    className?: string;
+    labelText?: string;
+    onFocus: () => void;
+    onBlur: () => void;
+};
+
+type DefaultProps = {
+    value: Props["value"];
+    onFocus: Props["onFocus"];
+    onBlur: Props["onBlur"];
+};
```

Comment:
I believe this could be `type DefaultProps = Pick<Props, 'value' | 'onFocus' | 'onBlur'>`, but I don't have a preference between the two

Reply:
Oh! Magical.  But certainly shortens these up and is probably a little more obvious that we're just copying those props exactly from `Props`.